lib/trino-filesystem/src/main/java/io/trino/filesystem/local/LocalFileSystem.java

```diff
@@ -219,6 +219,13 @@ public Set<Location> listDirectories(Location location)
         }
     }
 
+    @Override
+    public Optional<Location> createTemporaryDirectory(Location targetPath, String temporaryPrefix, String relativePrefix)
+            throws IOException
+    {
+        throw new IOException("Local file system does not support creating temporary directories");
```

Comment:
Why not?

Reply:
I didn't bother because this is just used for testing and none of the Hive tests use LocalFileSystem. We could implement if if needed.